openstax_accounts/tests.py

```diff
@@ -187,14 +186,25 @@ def test_stub(self):
         # users sorted by first name and last name
         self.assertEqual(
             sorted(users['items'],
-                   lambda a, b: cmp(a['username'], b['username'])), [
-                {'username': 'aaron', 'id': 1},
-                {'username': 'babara', 'id': 2},
-                {'username': 'caitlin', 'id': 3},
-                {'username': 'dale', 'id': 4},
-                {'username': 'earl', 'id': 5},
-                {'username': 'fabian', 'id': 6},
-                ])
+                   lambda a, b: cmp(a['username'], b['username'])),
+            [{'username': 'aaron', 'id': 1,
+              'first_name': "Aaron", 'last_name': "Andersen"},
+             {'username': 'babara', 'id': 2,
```

Comment:
I just checked accounts.  It returns something like this:

```
{u'total_count': 2, u'items': [
  {u'username': u'karenc4', u'first_name': u'karen', u'last_name': u'chan', u'id': 2, u'full_name': u'karen chan'}, 
  {u'username': u'user12345', u'id': 3}
]}
```

It doesn't always return all the fields.